Improve blog post descriptions (#328)
unicorn-utterances/unicorn-utterances: content/blog/documentation-driven-development/index.md

---
{
title: "A Better Way To Code: Documentation Driven Development",

> description: "Test driven development is often taught to improve a your dev workflow; I think works Documentation drive development (DDD) works better.",

Reviewer: ```suggestion
description: "Test Driven Development is often taught to improve a your workflow; I present Documentation Driven Development as an alternative approach.",
```
Noticed a small typo here - suggesting a slight rephrasing of this as well as fixing that.
